tests/utils/queryApiGateway.test.ts

```diff
@@ -31,9 +31,13 @@ const goodRequest = {
 
 // Test queryApiGateway()
 describe('Querying the API Gateway', () => {
-  const goodResponse = {
+  const goodResponse: Claim = {
```

Comment:
Let's call this something else since it's more like a nullish response. How about `mockedResponse`?